ci/setup-build-test-gdk.ps1

```diff
@@ -19,20 +19,18 @@ class TestProjectTarget {
         $this.test_repo_url = $test_repo_url
         $this.test_repo_relative_uproject_path = $test_repo_relative_uproject_path
         $this.test_project_name = $test_project_name
-        $this.test_gyms_version_path = $test_gyms_version_path
-        $this.test_env_override = $test_env_override
         
         # Resolve the branch to run against. The order of priority is:
         # envvar > same-name branch as the branch we are currently on > UnrealGDKTestGymVersion.txt > "master".
         $testing_repo_heads = git ls-remote --heads $test_repo_url $gdk_branch
         $test_gym_version = if (Test-Path -Path $test_gyms_version_path) {[System.IO.File]::ReadAllText($test_gyms_version_path)} else {[string]::Empty}
         if (Test-Path $test_env_override) {
-            $this.test_repo_branch = $test_env_override
+            $this.test_repo_branch = (Get-Item $test_env_override).value
         }
         elseif($testing_repo_heads -Match [Regex]::Escape("refs/heads/$gdk_branch")) {
             $this.test_repo_branch = $gdk_branch
         }
-        elseif(Test-Path $test_gym_version) {
+        elseif($test_gym_version -ne "") {
```

Comment:
very minor, you don't need -ne "" you can just:
`elseif($test_gym_version)`

Reply:
Hm, seems like it - however I find this a little bit more explicit, as we are setting it to an empty string above.
I could also reuse the empty string constant.